lib/rubocop/cop/rspec/pending_without_reason.rb

```diff
@@ -23,39 +23,61 @@ module RSpec
       #   xdescribe 'something' do
       #   end
       #
-      #   # bad
-      #   skip 'does something' do
+      #   # bad - when pending/skip block with string argument
+      #   #       but not inside an example
+      #   RSpec.describe 'something' do
+      #     pending 'does something'
       #   end
       #
-      #   # bad
-      #   it 'does something', :skip do
+      #   # good
+      #   it 'does something' do
+      #     pending 'reason'
+      #   end
+      #
+      #   # good
+      #   it 'does something', skip: 'reason' do
       #   end
       #
+      #   # good - when pending/skip not inside an example
+      #   FactoryBot.define do
+      #     factory :task do
+      #       pending
+      #     end
+      #   end
+      #
```

Comment:
```suggestion
```
I'd say it's not worth mentioning that this cop only inspects specs, and not factories, it's the way it should work with no doubt.